datahub-web-react/src/app/search/AdvancedSearchFilterOverallUnionTypeSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import { Select } from 'antd';
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { ANTD_GRAY } from '../entity/shared/constants';
+import { UnionType } from './utils/constants';
+
+type Props = {
+    unionType: UnionType;
+    onUpdate: (newValue: UnionType) => void;
+};
+
+const { Option } = Select;
+
+const StyledSelect = styled(Select)`
+    border-radius: 5px;
+    background: ${ANTD_GRAY[4]};
+    :hover {
+        background: ${ANTD_GRAY[4.5]};
```

Comment:
does this 4 vs 4.5 make a substantial visual difference?

Reply:
yes!